spec/factories.rb

```diff
@@ -1,5 +1,26 @@
-FactoryGirl.define do  factory :user do
-    
+FactoryGirl.define do  factory :job do
```

Comment:
Unnecessary spacing detected.<br>Block body expression is on the same line as the block start.